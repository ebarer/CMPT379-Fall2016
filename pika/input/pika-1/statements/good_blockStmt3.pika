exec {
	{
		var s := "me".
		print "They";.
		{
			{
				print "call";.
			}
			print s;.
		}
	}
	print "Mister";.
	print "Tibbs!", _n_.
}
