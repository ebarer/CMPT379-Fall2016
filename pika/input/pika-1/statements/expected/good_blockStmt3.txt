They call me Mister Tibbs!
